scripts/mail-all-activity.py

```diff
@@ -1,181 +0,0 @@
-#!/usr/bin/env python
```

Comment:
1. No, we can run this pretty much anywhere (as long as we have the production database downloaded), including the server from which the last bunch of emails was sent out.
2. Here: https://github.com/swcarpentry/amy/blob/develop/amy/settings.py#L62-L72